src/main/scala/com/typesafe/sbt/packager/archetypes/jlink/JlinkKeys.scala

```diff
@@ -11,6 +11,14 @@ private[packager] trait JlinkKeys {
   val jlinkBundledJvmLocation =
     TaskKey[String]("jlinkBundledJvmLocation", "The location of the resulting JVM image")
 
+  val jlinkModules = TaskKey[Seq[String]]("jlinkModules", "Modules to link")
+
+  val jlinkIgnoreMissingDependency =
+    TaskKey[((String, String)) => Boolean](
```

Comment:
Can you add some docs what the the function types represent in `(String, String) => Boolean)

Reply:
Done.